src/routes/accommodation.route.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import connect from 'connect-multiparty';
+import Accommodations from '../controllers/accommodation.controller';
+import AuthMiddleware from '../middlewares/auth.middleware';
+import AccommodationMiddleware from '../middlewares/accommodation.middleware';
+
+const router = express.Router();
+const connection = connect();
+
+router.post('/', AuthMiddleware.verifyToken, AccommodationMiddleware.isHost, connection, AccommodationMiddleware.validate, Accommodations.createAccommodation);
```

Comment:
Hi @shaazk Even the travel admin can add an accommodation. read well your PT story. Thank you.

Reply:
Let me fix it.